Containerfile.cert-manager

```diff
@@ -0,0 +1,44 @@
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_golang_1.22 AS builder
+ARG SOURCE_DIR="/go/src/github.com/openshift/jetstack-cert-manager"
+
+COPY cert-manager $SOURCE_DIR
+COPY cert-manager/LICENSE /licenses/
+
+ENV GO_BUILD_TAGS=strictfipsruntime,openssl
+ENV GOEXPERIMENT=strictfipsruntime
+ENV CGO_ENABLED=1
+ENV GOFLAGS=""
+
+RUN cd $SOURCE_DIR/cmd/acmesolver && go build -o $SOURCE_DIR/_output/acmesolver -ldflags '-w -s' -tags ${GO_BUILD_TAGS} main.go
+RUN cd $SOURCE_DIR/cmd/cainjector && go build -o $SOURCE_DIR/_output/cainjector -ldflags '-w -s' -tags ${GO_BUILD_TAGS} main.go
+RUN cd $SOURCE_DIR/cmd/controller && go build -o $SOURCE_DIR/_output/controller -ldflags '-w -s' -tags ${GO_BUILD_TAGS} main.go
+RUN cd $SOURCE_DIR/cmd/webhook && go build -o $SOURCE_DIR/_output/webhook -ldflags '-w -s' -tags ${GO_BUILD_TAGS} main.go
+
+FROM registry.redhat.io/rhel9-2-els/rhel:9.2-1362
+
+ARG RELEASE_VERSION
```

Comment:
nit: add comments about the value generation for the args.

Reply:
comment incorporated.